src/app/(record)/notice/[id]/page.tsx

```
@@ -21,7 +21,7 @@ export default async function Page({ params }: PageProps) {
       <h3 className="text-[20px] font-bold">{data.title}</h3>
       <span className="text-[14px] text-slate-600">{formatDate(data.createdAt)}</span>
       <hr />
-      <p className="mb-16 mt-10">{data.content}</p>
+      <div className="prose mb-16 mt-10" dangerouslySetInnerHTML={{ __html: data.content }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)